### PR TITLE
(fix) multi folders in workspace

### PR DESCRIPTION
**Pull request by g-plane**
Currently, Svelte Language Server doesn't work well when there're multiple folders in a workspace. This PR fixes it.

Hope to be merged.

### CODE REVIEW

File: packages/language-server/src/server.ts [hunk 1]
```diff
@@ -86,9 +86,9 @@ export function startServer(options?: LSOptions) {
     let sveltePlugin: SveltePlugin = undefined as any;
 
     connection.onInitialize((evt) => {
-        const workspacePath = urlToPath(evt.rootUri || '') || '';
-        Logger.log('Initialize language server at ', workspacePath);
-        if (!workspacePath) {
+        const workspacePaths = evt.workspaceFolders?.map(folder => folder.uri.toString()) ?? [];
```
Comment (jasonlyu123) [inline packages/language-server/src/server.ts:89]: For potential backward compatible and for LSP clients that don't support multi-root-workspaces can you make it fallback to the old behavior?

**Review by jasonlyu123 — CHANGES_REQUESTED** [review]
If I understand correctly you are trying to support [multi-root-workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces)?

File: packages/language-server/src/server.ts (hunk 1 above)
Comment (g-plane) [reply packages/language-server/src/server.ts:89]: Fallback to `evt.rootUri`?

Comment (jasonlyu123) [reply packages/language-server/src/server.ts:89]: Yes.

**Comment by g-plane** [comment]
Are there any other problems which should be addressed?

**Review by jasonlyu123 — APPROVED** [review]

**Review by dummdidumm — APPROVED** [review]
👍

**Pull request merged**